docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,161 @@
+---
+lang: en
+title: 'Inside a Loopback Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-Loopback-Application.html
+---
+
+# APIs in a digital era
+
+Digital transformation is changing how data and services are shared across Web/Mobile/IoT platforms.
+APIs allow easy and secure access to digital resources and capabilities of an organization by internal and enternal parties.
+
+Developers are looking out for more productive ways of creating and consuming APIs.
+
+Consider a typical `Online Shopping` site. First the UI requirements are gathered. Customers would be provided a home page, a product list, a shopping cart and an order history page.
+
+<img align="right" src="/docs/site/imgs/mobile-shopping-app.png" alt="Shopping App"
+	title="Digital Era" width="300" height="600" />
```

Comment:
I'm not sure if it's a good idea to align the image to `right`, especially when it's rendered on a mobile browser.

Reply:
Yes, aligned images usually dont look good in mobile browsers; but the amazon app image is  vertically stretched. If it is placed in the center that would look more odd.

Reply:
the amazon image is now replaced with a prototype image and the alignment is changes as per what you had suggested.